examples/nirfsa/getting-started-spectrum.py

```diff
@@ -64,86 +65,60 @@
+            session_name=SESSION_NAME, resource_name=RESOURCE, option_string=OPTIONS
         )
     )
     vi = init_response.vi
 
-    raise_if_error(
-        client.ConfigureRefClock(
-            nirfsa_types.ConfigureRefClockRequest(
-                vi=vi,
-                clock_source_mapped=nirfsa_types.RefClockSource.REF_CLOCK_SOURCE_ONBOARD_CLOCK,
-                ref_clock_rate=10e6,
-            )
+    client.ConfigureRefClock(
+        nirfsa_types.ConfigureRefClockRequest(
+            vi=vi,
+            clock_source_mapped=nirfsa_types.RefClockSource.REF_CLOCK_SOURCE_ONBOARD_CLOCK,
+            ref_clock_rate=10e6,
         )
     )
-    raise_if_error(
-        client.ConfigureReferenceLevel(
-            nirfsa_types.ConfigureReferenceLevelRequest(vi=vi, reference_level=0)
-        )
+    client.ConfigureReferenceLevel(
+        nirfsa_types.ConfigureReferenceLevelRequest(vi=vi, reference_level=0)
     )
-    raise_if_error(
-        client.ConfigureAcquisitionType(
-            nirfsa_types.ConfigureAcquisitionTypeRequest(
-                vi=vi, acquisition_type=nirfsa_types.AcquisitionType.ACQUISITION_TYPE_SPECTRUM
-            )
+    client.ConfigureAcquisitionType(
+        nirfsa_types.ConfigureAcquisitionTypeRequest(
+            vi=vi, acquisition_type=nirfsa_types.AcquisitionType.ACQUISITION_TYPE_SPECTRUM
         )
     )
-    raise_if_error(
-        client.ConfigureSpectrumFrequencyStartStop(
-            nirfsa_types.ConfigureSpectrumFrequencyStartStopRequest(
-                vi=vi, start_frequency=990e6, stop_frequency=1010e6
-            )
+    client.ConfigureSpectrumFrequencyStartStop(
+        nirfsa_types.ConfigureSpectrumFrequencyStartStopRequest(
+            vi=vi, start_frequency=990e6, stop_frequency=1010e6
         )
     )
-    raise_if_error(
-        client.ConfigureResolutionBandwidth(
-            nirfsa_types.ConfigureResolutionBandwidthRequest(vi=vi, resolution_bandwidth=10e3)
-        )
+    client.ConfigureResolutionBandwidth(
+        nirfsa_types.ConfigureResolutionBandwidthRequest(vi=vi, resolution_bandwidth=10e3)
     )
-    spectral_lines_response = raise_if_error(
-        client.GetNumberOfSpectralLines(nirfsa_types.GetNumberOfSpectralLinesRequest(vi=vi))
+    spectral_lines_response = client.GetNumberOfSpectralLines(
+        nirfsa_types.GetNumberOfSpectralLinesRequest(vi=vi)
     )
-
-    read_response = raise_if_error(
-        client.ReadPowerSpectrumF64(
-            nirfsa_types.ReadPowerSpectrumF64Request(
-                vi=vi,
-                timeout=10.0,
-                data_array_size=spectral_lines_response.number_of_spectral_lines,
-            )
+    read_response = client.ReadPowerSpectrumF64(
+        nirfsa_types.ReadPowerSpectrumF64Request(
+            vi=vi,
+            timeout=10.0,
+            data_array_size=spectral_lines_response.number_of_spectral_lines,
         )
     )
-
+    check_for_warning(read_response, vi)
```

Comment:
There was a blank line between the read and the comment block. You should keep 1 blank line after the warning check.